apps/docs/.spellcheck.yml

```diff
@@ -0,0 +1,33 @@
+matrix:
```

Comment:
Little bit confused at how this matrix is being implemented? Is this not just a single job?